docs/coding-conventions.md

```diff
@@ -0,0 +1,89 @@
+# Coding conventions
```

Comment:
We have bash scripts and also golang scripts. Can you maybe add that separation like in Kubernetes example [1]?

[1] https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md#code-conventions

Reply:
Right, I forgot about that and will add something. But for now I wanted to focus more on the golang conventions.